vpc-template/Makefile

```diff
@@ -0,0 +1,8 @@
+.DEFAULT_GOAL := vpc
+
+check:
+	flake8 stack/
+	isort --recursive --check-only --diff stack/
+
+vpc:
+	python -c 'import stack' > cloudformation-vpc-template.json
```

Comment:
If we're generating cloudformation-vpc-template.json at runtime, maybe it doesn't need to be added to git.  It could be a candidate for .gitignore.

Reply:
Personally I like having this in the repo, so we have a record of changes to the stack itself when those might not be obvious from the Python code; e.g., see this bug fix: https://github.com/caktus/fabulaws/pull/80/commits/b2d8703b0fa19ce68c679ed1d6ee36258f823891

It also makes it unnecessary for a user of this stack to get a local development environment set up, since we can just link to the file in git.